.github/wiki/Fastlane.md

```diff
@@ -1,150 +1,146 @@
-Fastlane is a part of the automation tool for the development and release process of the project. This document will mention some crucial parts of the pipeline.
+Fastlane is a part of the automation tool for the development and release process of the project. By converting to Fastlane.swift, we can now write our configuration using Xcode in Swift - the language we know and love from the world of iOS development. This document will mention some crucial parts of the pipeline. 
 
 <img src="assets/images/infrastructure/fastlane/fastlane.png" width=300>
 
 ## Fastfile
 
 ### Build and test
 
-The lane `build_and_test` helps build and test the application with the configuration of the test scheme and test target defined in the `Constants.rb` file.
+The lane `buildAndTestLane` helps build and test the application with the configuration of the test scheme and test target defined in the `Constant.swift` file.
```

Comment:
FYI, we can safely remove the suffix `Lane` for all lanes :D.
```suggestion
The lane `buildAndTest` helps build and test the application with the configuration of the test scheme and test target defined in the `Constant.swift` file.
```

Reply:
Feel free to remove all the rest lanes, which still contain the suffix `Lane` in both the description and executing command🙏  Thank youu.